Razmik_Kirakosyan/python/fibonacci/fibon_num.py

```diff
@@ -0,0 +1,17 @@
+def print_fibonacci (n):
+	x=0
+	y=1
+	
+	if n == 1: 
+		print (x)
+	if n == 2:
+		print (y)
+	if n > 2:
+		for i in range(2,n):
+			z=x+y
+			x=y
+			y=z
+			
+		print(z)
+n = int(input())
+print_fibonacci(n)
```

Comment:
- No any info what to input from user side
- for 0 doesn't work correctly
![image](https://user-images.githubusercontent.com/80816144/119725188-b1063f00-be80-11eb-94f3-fb3800dee425.png)
- no validation for invalid values
![image](https://user-images.githubusercontent.com/80816144/119725254-cd09e080-be80-11eb-92be-047250e05d03.png)
- fix style with pep8